myhpom/migrations/0003_user_details.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+import os, json
+from django.db import migrations, models
+import django.db.models.deletion
+from django.conf import settings
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ('myhpom', '0002_states'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='UserDetails',
```

Comment:
You expressed the need to not reuse UserProfile and I agree...the less we're entangled with the hydroshare codebase the better!

Reply:
+1